.stickler.yml

```diff
@@ -0,0 +1,13 @@
+# add the linters you want stickler to use for this project
+linters:
+  eslint:
+    # indicate where is the config file for stylelint
+    config: './.eslintrc.json'
+files:
+    ignore:
+        - 'src/*'
```

Comment:
Ignoring the `src` directory is the same with not adding a linter since that's where all the code is.
Please, delete this. :relaxed: 
Let's use exactly what is here (https://github.com/microverseinc/linters-config/blob/master/react-redux/.stickler.yml) instead.